src/edgeDebugAdapter.ts

```diff
@@ -329,10 +344,14 @@ export class EdgeDebugAdapter extends CoreDebugAdapter {
             const edgeProc = fork(getEdgeSpawnHelperPath(), [edgePath, ...edgeArgs], options);
```

Comment:
I synced with Rob and we use the spawnHelper so that the Edge process doesn't get killed automatically when the debug adapter is killed. If we can find to make this option work: https://nodejs.org/api/child_process.html#child_process_options_detached or another way to achieve this behavior, or we realize that it doesn't really affect Edge, then we would be able to remove the spawnHelper and spawn the process directly from here, making the code simpler

Reply:
The document says `Specific behavior depends on the platform` so I feel that that option is risky unless we do intensive testing on all the platforms. Despite simpler, I don't think introducing such code change at this moment is worthy, since it is changing a way that we have already validated for so long.

Reply:
What platforms are you thinking of?

Reply:
That is a good point. I will see what I can do.